### PR TITLE
Add ability to add / remove org-level webhooks

### PR DESCRIPTION
**Pull request by jali-clarke**
This PR adds the ability to add / remove org-level webhooks alongside repo-level webhooks.  Some refactoring was done due to resultant code duplication.  It also removes the `-p / --password` cli option since it was not being used anywhere after having been parsed.

edit:  Lots of refactoring of the cli code too in order to make it testable - codecov complained about code coverage.

edit the second:  Added integration-y tests for cli actions

### CODE REVIEW

File: tests/cli/test_parsers.py [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+                    "you are removing hooks from. Useful when the "
+                    "user in settings is not the administrator of "
+                    "your organization."
+                )
+            ),
+            call(
+                'org_name',
+                help="The login name of the organization."
+            )
+        ]
```
Comment (markstory) [inline tests/cli/test_parsers.py]: These tests seem to duplicate most of the logic in the commands. Is there a way we could use `responses` to instead do an integration test?

Comment (jali-clarke) [reply tests/cli/test_parsers.py]: Thanks for your feedback Mark - can you give me an example of what you might see as an integration test flow?  In my view, it seems like we'd need something like a mock GitHub api server.

**Comment by jali-clarke** [comment]
@markstory added the integration tests in place of those particular unit tests

**Comment by jali-clarke** [comment]
@markstory added the integration tests in place of those particular unit tests

**Comment by markstory** [comment]
Thanks. I'll take a look. I'm not sure I want to support organization level hooks at this point. It doesn't fit well with the future of the project which will be github-app only. The OAuth application support is likely to be deprecated and removed as soon as I have the time to do so.

**Comment by jali-clarke** [comment]
> The OAuth application support is likely to be deprecated and removed as soon as I have the time to do so.

I see!  I can close this PR if that's the case.

**Pull request closed without merging**